authors/apps/authentication/backends.py

```diff
@@ -1,10 +1,46 @@
-# import jwt
-#
-# from django.conf import settings
-#
-# from rest_framework import authentication, exceptions
-#
-# from .models import User
+import jwt
+
+from django.conf import settings
+
+from rest_framework import authentication, exceptions
+
+from .models import User
 
 """Configure JWT Here"""
+class JWTAuthentication(authentication.BaseAuthentication):
+    authentication_header_prefix = 'token'
+    def authenticate(self, request):
+        request.user = None
+        
+        auth_header = authentication.get_authorization_header(request).split()
+        auth_header_prefix = self.authentication_header_prefix.lower()
+
+        if not auth_header:
```

Comment:
@VivianDoreen You can refactor this code and reduce the number of if's that are used. Look into this.